src/provider/utils/Timer.ts

```diff
@@ -1,21 +1,27 @@
 export class Timer {
     private readonly _duration: number;
-    private _timeoutHandle: number | undefined;
+    private _timeoutHandle: any | undefined;
```

Comment:
I'm sure there is a type for this somewhere in the standard types right?

Reply:
Ah, meant to change that back. Two minutes!